components/org.wso2.carbon.identity.user.endpoint/src/test/java/org.wso2.carbon.identity.user.endpoint.Util/HandleExceptionsTest.java

```diff
@@ -65,6 +67,33 @@ public void testConflictException() {
         assertEquals(conflictException.getResponse().getStatus(), 409);
     }
 
+    @Test
+    public void testHandleConflictException(){
 
-}
+        ConflictException conflictException = new ConflictException();
+        assertEquals(conflictException.getResponse().getStatus(),409);
+        String errorCode = "UNR-10002";
+        String errorMessage = "Test Error";
+        try {
+            Utils.handleConflict(errorMessage, errorCode);
+        } catch (ConflictException exception) {
+            assertEquals(exception.getResponse().getStatus(),409);
+            assertEquals(exception.getMessage(),errorMessage);
+        }
+    }
 
+    @Test
+    public void testHandleNotAcceptableException(){
+
+        NotAcceptableException notAcceptableException = new NotAcceptableException();
+        assertEquals(notAcceptableException.getResponse().getStatus(),406);
+        String errorCode = "UNR-10002";
+        String errorMessage = "Test Error";
+        try {
+            Utils.handleNotAcceptable(errorMessage, errorCode);
+        } catch (NotAcceptableException exception) {
+            assertEquals(exception.getResponse().getStatus(),406);
```

Comment:
formatting issues in line 95 and 96

Reply:
Fixed: f929c4ac1bd444a80c901a7c4281882c7728403a